risc0/zkvm/src/guest/env/mod.rs

```diff
@@ -78,26 +79,27 @@ use alloc::{
     vec,
 };
 
+use crate::{
+    sha::{
+        rust_crypto::{Digest as _, Sha256},
+        Digest, Digestible,
+    },
+    Assumptions, MaybePruned, Output,
+};
```

Comment:
can you move this down to near the `self` location? Basically we try to group uses by:

* std
* external
* internal